lib/types/CredentialIssuance.types.ts

```diff
@@ -48,57 +46,60 @@ export type EncodeJsonAsURIOpts = { uriTypeProperties?: string[]; arrayTypePrope
 
 export type DecodeURIAsJsonOpts = { requiredProperties?: string[]; arrayTypeProperties?: string[] };
 
-export interface JWK {
```

Comment:
this is a subset present in the specs. I think it's better than a generic unknown